docs/content/Logging.md

```diff
@@ -6,34 +6,35 @@ Logging
 
 Druid nodes will emit logs that are useful for debugging to the console. Druid nodes also emit periodic metrics about their state. For more about metrics, see [Configuration](Configuration.html). Metric logs are printed to the console by default, and can be disabled with `-Ddruid.emitter.logging.logLevel=debug`.
 
-Druid uses [log4j](http://logging.apache.org/log4j/2.x/) for logging, and console logs can be configured by adding a log4j.xml file. Add this xml file to your classpath if you want to override default Druid log configuration.
-
-An example log4j.xml file is shown below:
+Druid uses [log4j2](http://logging.apache.org/log4j/2.x/) for logging. Logging can be configured with a log4j2.xml file. Add the path to the directory containing the log4j2.xml file (eg a config dir) to your classpath if you want to override default Druid log configuration. Note that this directory should be earlier in the classpath than the druid jars. The easiest way to do this is to prefix the classpath with the config dir. For example, if the log4j2.xml file is in config/_common:
 
+```bash
+java -Xmx512m -Duser.timezone=UTC -Dfile.encoding=UTF-8         \
+     -Ddruid.realtime.specFile=examples/indexing/wikipedia.spec \
+     -classpath "config/_common:config/realtime:lib/*"          \
+     io.druid.cli.Main server realtime
 ```
-<?xml version="1.0" encoding="UTF-8" ?>
-<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
-
-<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/">
 
-  <appender name="ConsoleAppender" class="org.apache.log4j.ConsoleAppender">
-    <layout class="org.apache.log4j.PatternLayout">
-      <param name="ConversionPattern" value="%d{ISO8601} %-5p [%t] %c - %m%n"/>
-    </layout>
-  </appender>
+Note the "-classpath" in this example has the confid dir before the jars under lib/*.
 
-  <!-- ServerView-related stuff is way too chatty -->
-  <logger name="io.druid.client.BatchServerInventoryView">
-    <level value="warn"/>
-  </logger>
-  <logger name="io.druid.curator.inventory.CuratorInventoryManager">
-    <level value="warn"/>
-  </logger>
+An example log4j2.xml ships with Druid under config/_common/log4j2.xml, and a sample file is also shown below:
 
-  <root>
-    <priority value="info" />
-    <appender-ref ref="ConsoleAppender"/>
-  </root>
-
-</log4j:configuration>
+```
+<?xml version="1.0" encoding="UTF-8" ?>
+<Configuration status="WARN">
+  <Appenders>
+    <Console name="Console" target="SYSTEM_OUT">
+      <PatternLayout pattern="%d{ISO8601} %p [%t] %c - %m%n"/>
+    </Console>
+  </Appenders>
+  <Loggers>
+    <Logger name="io.druid.curator.inventory.CuratorInventoryManager" level="warn"/>
+    <Logger name="io.druid.client.BatchServerInventoryView" level="warn"/>
+    <Logger name="io.druid.client.ServerInventoryView" level="warn"/>
+    <Logger name ="com.metamx.http.client.pool.ChannelResourceFactory" level="warn"/>
```

Comment:
These were in the original doc as well as in the Performance FAQ, so it seemed they were likely candidates for tuning. Should I remove them (from here and the xml file) anyway?